pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImpl.java

```diff
@@ -1036,6 +1037,8 @@ private void doSubscribeTopicPartitions(Schema<T> schema,
                         synchronized (pauseMutex) {
                             if (paused) {
```

Comment:
If we create a new consumer with `pause` state, we don't need to pause it again.

Reply:
I think maybe we even can remove the `synchronized (pauseMutex)`

Reply:
> I think maybe we can remove the `synchronized (pauseMutex)` too

Not really, if the pause state changes between after 1038 and before line 1043, pause state of this newConsumer will be wrong.

> If we create a new consumer with `pause` state, we don't need to pause it again.

Yes, but current implementation of `newConsumer.pause()` and `newConsumer.resume()` is idempotent and handles the case when it's called repeatedly. I think the code is more clear this way. WDYT? @aloyszhang

Reply:
make sense to me